lib/web/apiserver.go

```diff
@@ -833,6 +833,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/databases", h.WithClusterAuth(h.awsOIDCListDatabases))
 	h.GET("/webapi/scripts/integrations/configure/listdatabases-iam.sh", h.WithLimiter(h.awsOIDCConfigureListDatabasesIAM))
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/deployservice", h.WithClusterAuth(h.awsOIDCDeployService))
+	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/deploydatabaseservices", h.WithClusterAuth(h.awsOIDCDeployDatabaseServices))
```

Comment:
Any reason we're introducing a new API specifically for database services, rather than either updating existing "deployservice" to allow deploying multiple, or at least having a separate generic "deployservices"?

Reply:
The `deployservice` API tries to be generic for every service, but I don't think that was going to work.
When I was trying to re-use it, it didn't felt very flexible. And that's a considerable small change: deploy multiple database services instead of one.
If we changed the API, we would also need to change the UI.

This new API is more specific, and is able to handle the single RDS enrollment anyway, we just need to change the UI.

After changing the UI, the previous endpoint can be removed.